css/style.css

```diff
@@ -0,0 +1,229 @@
+* {
+  box-sizing: border-box;
+}
+
+.row {
+  background: url(https://images.unsplash.com/photo-1508848484850-f696c839aabd?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=e4a425b35d743e14ed00824b39731186&auto=format&fit=crop&w=1600&q=80);
+  background-size: cover;
+  background-position: right bottom;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  width: 100%;
+}
+
+.row::after {
+  content: '';
+  clear: both;
+  display: block;
+}
+
+[class*='col-'] {
+  float: left;
+  padding: 15px;
+}
+
+html {
+  font-family: 'Kalam', sans-serif;
+}
+
+html,
+body {
+  margin: 0;
+  height: 100%;
+}
+
+.header {
+  background-color: #64827a;
+  padding: 15px;
+}
+
+main .row {
+  padding: 15px;
+  text-align: center;
+}
+
+.pizza-item section {
+  padding: 15px;
+  border: 2px solid #fff;
+}
+
+.pizza-info section {
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  background-color: #64827a;
+  opacity: 0.7;
+  filter: alpha(opacity=60); /* For IE8 and earlier*/
+}
+
+.pizza-img {
+  padding: 25px;
+  background-color: #64827a;
+  opacity: 0.7;
+  filter: alpha(opacity=60); /* For IE8 and earlier*/
+}
+
+.header ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  font-size: 18px;
+  /*   opacity: 0.6;
+filter: alpha(opacity=60); /* For IE8 and earlier */
+}
+
+.header li {
+  padding: 6px;
+  margin: ;
+  /*     background-color: ; */
+  /*   float: left; */
+}
+
+.header li a {
+  display: flex;
+  justify-content: center;
+  color: #edb786;
+  padding: 10px;
+  text-decoration: none;
+  border: 2px solid;
+}
+
+li.active > a {
+  text-align: right;
+  color: orangered;
+  font-weight: bold;
+}
+
+.header li:hover:not(.active) {
+  cursor: default;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: center;
+}
+
+#add-pizza {
+  background-color: #64827a;
+  background-color: orangered;
+  cursor: pointer;
+  padding: 15px;
+  color: #f5e0ce;
+  display: flex;
+  justify-content: center;
+  font-family: 'Kalam', sans-serif;
+  letter-spacing: 0.2em;
+  font-size: 18px;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  /*   opacity: 0.6;
+filter: alpha(opacity=60); /* For IE8 and earlier */
+}
+.footer {
+  background-color: #64827a;
+  color: #edb786;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  font-size: 1.1em;
+  padding: 15px;
+}
+
+/* For mobile phones: */
+[class*='col-'] {
+  width: 100%;
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
It does not look like generated. Try to avoid it. Read about DRY.

I prefer to use flex partially and do not develop grid frameworks.